app/views/albums/show.html.erb

```diff
@@ -23,7 +26,7 @@
   </p>
 </div>
 
-<% if logged_in_as_admin? || @album.user == current_user || @album.shared? then %>
+<% if logged_in_as_site_admin? || logged_in_as_admin_of?(@album.circle) || @album.user == current_user || @album.shared? then %>
```

Comment:
Is the `then` keyword necessary in this view?

Reply:
The then is never needed, it is just my habit to write it there